backend/models.py

```diff
@@ -1,15 +1,15 @@
 import enum
-from datetime import datetime
 from typing import List, Union, Optional, NamedTuple
+from datetime import datetime
 
 from sqlalchemy import (
-    select, Enum, DateTime,
+    select, Enum, DateTime, Date,
     Boolean, Column, ForeignKey, 
     Integer, String, Float)
```

Comment:
I'm not sure why, but these imports did not work well on my machine. Maybe you would like to rewrite them to:

```python
from sqlalchemy import select, Column, ForeignKey
from sqlalchemy.types import Enum, DateTime, Date, Boolean, Integer, String, Float
```

Also, you probably would like to create a migration for these changes :wink:

Reply:
> did not work well on my machine

What was exactly the problem?

Reply:
I created a migration with alembic and tried to apply it. That failed because the it (the DB or sqlalchemy) claimed that it did not know the datatype `contenttype`.
After changing the imports like that and re-creating the migration, it was successfully applied. I admit, it's a bit weird :thinking:

Reply:
Did you generate a new migration on top of the second (the one I commited for previous version of traceability)? And did you use command `alembic revision --autogenerate`? For some reason I get migrations with empty methods🤔

Reply:
Yes, that was my approach and the command actually created a non-empty migration.

Now that I think about it, it would probably make sense to replace the existing migration :thinking: 

Did you run the command in the  repo's root directory? That might also impact the behavior of alembic

Reply:
> Did you run the command in the repo's root directory

I did

So, I finally had to use radical measures which was to drop all tables, move models to state before this PR, then upgrade to second migration, then insert new models and generate (now successfully) new migration, add some statements there to create enums (it's almost 2022 soon, why alembic can't create enums on its own?🤔) and finally apply this migration.

I've committed it (along with changes in models imports). I can't say for sure that migration worked correctly by running tests, because they use other database which is recreated all the time. But at a quick glance all tables are present and frontend seems to be working well with backend